### ISSUE TITLE
Suppressor types and their damage reduction

### ISSUE THREAD

**Issue by CatWithFourEyes** [CleverRaven/Cataclysm-DDA#65651]
### Is your feature request related to a problem? Please describe.

Suppressors currently have very different damage penalty stats (some silencers have it, some don't) that don't have a rationale, as the impact of suppressors on bullet speed isn't significant. In some cases, the speed of the bullet even increases.
The most notable case is the huge damage penalty of the built-in MP5SD suppressor, which is 7 points, while improvised improvised suppressors have a smaller penalty, which is rather strange.

### Solution you would like.

Remove damage reduction from factory-made suppressors, while keeping a small penalty for improvised ones (or increasing the impact of the latter on shooting accuracy, simulating possible bullet trajectory deviations).

### Describe alternatives you have considered.

1. Don't touch it.
2. More consistent damage reducing beetween suppressors

### Additional context

https://arxiv.org/ftp/arxiv/papers/1802/1802.05748.pdf - proof for .300WinMag rifle, deviations in muzzle velocity in 10-30 fps in both directions
https://youtu.be/kQxLmJOOAIY - another one, .308 rifle
https://youtu.be/70p3fvb7Edo - different guns, overall very small increased muzzle velocity

**Comment by GuardianDll** [comment]
Sure, feel free to tweak a stats according to real specs

**Comment by tenmillimaster** [comment]
Suppressors can affect damage in several ways. 

Baffle suppressors -
A well performing baffle type suppressor can slightly increase the velocity of a projectile. This includes wire mesh type suppressors. It was my assumption that this was negligible by the time we got to CDDA damage numbers. Feel free to audit this.

Wipe suppressors - any Wiped suppressor will affect the velocity of a projectile negatively. This includes suppressor using non-rubber media to drag on the projectile (like brass wool, for example). The joke-item bottle suppressor is in this category. Many are designed to take a load from supersonic to subsonic.

Flow-through suppressors- I am uncertain at this point how these affect projectile velocity. We don't have any of these in game currently.

Bleed-offs: there is a special case here, where a barrel is ported to bleed off propellant gas into the suppressor body to reduce the velocity of a projectile to subsonic. This includes the MP5SD. Any case where the barrel is ported like this will by design reduce projectile velocity.


In short, feel free to audit the damage numbers but care must be taken to consider suppressor+host design, and since we don't have damage stats per caliber for suppressors, it will be difficult to reach correct numbers in every case.

**Comment by CatWithFourEyes** [comment]
Did we have any wiped suppressors, except homemade suppressor (item id "crafted_suppressor") (bottle suppressor... it's hard to think about it other way then "Survivor: Hey, i see this in a film/read in novel, maybe it's work!", honestly)? As far as i know, they have very limited number of shots, before starting "leak" sound, from wearing wipes, but somewhat more effective due "sealing"  (also ingame, homemade suppressor on par with factory one), metallic wools also must "burn" quickly. 

About MP5SD i relied on the following video and measurements in it (yes, not ideal proof, but still better than different forums chitchat without measurements): https://youtu.be/jYkq6y-CjOM
on 124gr bullet it was 995fs and 273ft/lbs (=370.1J). Square root from 370 = 19.23 (like .38 SW, relies upon GAME_BALANCE.md recomendations) so ~19 dmg for FMJ and 21 for JHP's seems reasonable for me, and -7 penalty still to big, and -4/-5 points will be more accurate.

**Comment by tenmillimaster** [comment]
> Did we have any wiped suppressors, except homemade suppressor (item id "crafted_suppressor")  

No, we do not have any (real) wiped suppressors in game. That's not to say we couldn't, there are indeed a handful of wiped designs out there, on the civilian market no less, for various reasons. The station 6, the pillbox, aurora, alongside other even more niche products...

>-4/-5 points will be more accurate 

Sure, that sounds about right.

**Comment by tenmillimaster** [comment]
I don't think wiped suppressors can currently be handled in JSON. While I wouldn't mind weird, ultra quiet suppressors that only last for a few shots and don't blow up the entire suppressor, that'd require additional faults to be coded in.

**Comment by CatWithFourEyes** [comment]
> don't blow up the entire suppressor

Is it so big problem? They can be just unrepairable, and breaking like "blowed up" improvised ones. At least as temporary option. I don't think they can be restored to "mint new" with some duct tape and random rubber (after this, it's essentialy "homemade suppressor...).

**Comment by tenmillimaster** [comment]
It depends on the design. A real (or sufficiently well made) wiped suppressor needs to just have some fault and be repaired with rubber sheet of some sort. The homemade ones can blow up as they already should be.
